website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -6416,6 +6421,68 @@ with pipeline as p:
 
 4. After the job has been submitted to the Beam runner, shutdown the expansion service by terminating the expansion service process.
 
+#### 13.2.3. Using cross-language transforms in a Go pipeline
+
+If a Go-specific wrapper for a cross-language is available, use that; otherwise, you have to use the
+lower-level [CrossLanguage](https://pkg.go.dev/github.com/apache/beam/sdks/go/pkg/beam#CrossLanguage)
+function to access the transform.
+
+**Expansion Services**
+
+The Go SDK does not yet support automatically starting an expansion service. In order to use
+cross-language transforms, you must manually start any necessary expansion services on your local
+machine and ensure they are accessible to your code during pipeline construction; see more at
```

Comment:
We should also add something analogous to the following in Python section.

"Make sure the transform you're trying to use is available and can be used by the expansion service. For Java, make sure the builder and registrar for the transform are available in the classpath of the expansion service."

Reply:
I actually had this before, but removed it in favor of the current approach linking to the "how to create" section, in order to avoid putting details about Java functionality in the Go section.